pkg/noderesourcetopology/score_test.go

```diff
@@ -0,0 +1,294 @@
+package noderesourcetopology
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+	"testing"
+
+	topologyv1alpha1 "github.com/k8stopologyawareschedwg/noderesourcetopology-api/pkg/apis/topology/v1alpha1"
+	listerv1alpha1 "github.com/k8stopologyawareschedwg/noderesourcetopology-api/pkg/generated/listers/topology/v1alpha1"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	intstr "k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/klog/v2"
+	framework "k8s.io/kubernetes/pkg/scheduler/framework/v1alpha1"
+)
+
+// return the name of the node with the highest score
+func findMaxScoreNode(nodeToScoreMap map[string]int64) string {
+	var max int64 = 0
```

Comment:
this comment mostly about a style, why not to declare & initialize it the same way as electedNode, e.g. max := int64(0)